src/indexer/broker/src/server/sse.rs

```diff
@@ -0,0 +1,59 @@
+use std::{convert::Infallible, sync::Arc};
```

Comment:
This is useful as an example, but how about removing it from getting compiled?

Reply:
This is already the case. Check `server.rs`. I use feature flags to compile this or not. I just configured docker to not compile this by default.